### PR TITLE
cd, cmd, dir: add Thai translation

### PR DESCRIPTION
**Pull request by nostprumprajee**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

**Comment by navarroaxel** [comment]
@nostprumprajee are you working in this?

**Comment by navarroaxel** [comment]
@2KAbhishek @9tykeshav do you like to help here?

**Comment by 9tykeshav** [comment]
@navarroaxel hey there ! , What way would you like us to help ? 
(are you mistaking us with thai ? , pretty sure we both have contributed for Hindi translation , i do not have any experience in thai language). 
I'll be happy to help if i can !

**Comment by marchersimon** [comment]
@navarroaxel should we just merge this page as is? We only have one Thai page so far, so we won't be able to find anyone who could help.

**Comment by kbdharun** [comment]
Hi, I have reopened this PR. I will request reviews from native speakers who helped me out with #8585 during Hacktoberfest last year.

@MasterIceZ @Leomotors Can you review this PR and suggest translations for the placeholders? Thanks in advance.

> Once the placeholder is translated and other changes are reviewed I will force merge this PR as the PRs author has signed the CLA previously before December 2022.

File: pages.th/window/cd.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> แสดงชื่อหรือเปลี่ยนไดเร็กทอรีการทำงานปัจจุบัน
+> ข้อมูลเพิ่มเติม: <https://docs.microsoft.com/windows-server/administration/windows-commands/cd>.
+
+- ไปที่ไดเร็กทอรีในไดรฟ์เดียวกัน:
+
+`cd {{path/to/directory}}`
```
[the remark this answers is not in the document]
Comment (kbdharun) [reply pages.th/window/cd.md:8]:
```suggestion
`cd {{ทาง/ไป/สารบบ}}`
```

File: pages.th/window/cd.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`cd`
+
+- ไปที่พาเรนต์ของไดเร็กทอรีปัจจุบัน:
+
+`cd ..`
+
+- ไปที่ไดเร็กทอรีที่อยู่ต่างไดรฟ์:
+
+`cd {{path/to/directory}} /d`
```
Comment (kbdharun) [inline pages.th/window/cd.md:20]:
```suggestion
`cd {{ทาง/ไป/สารบบ}} /d`
```

File: pages.th/window/cd.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# cd
+
+> แสดงชื่อหรือเปลี่ยนไดเร็กทอรีการทำงานปัจจุบัน
+> ข้อมูลเพิ่มเติม: <https://docs.microsoft.com/windows-server/administration/windows-commands/cd>.
```
Comment (kbdharun) [inline pages.th/window/cd.md:4]:
```suggestion
> ข้อมูลเพิ่มเติม: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cd>.
```

File: pages.th/window/cmd.md [hunk 4]
```diff
@@ -0,0 +1,36 @@
+# cmd
+
+> ตัวแปลคำสั่งของ Windows.
+> ข้อมูลเพิ่มเติม: <https://docs.microsoft.com/windows-server/administration/windows-commands/cmd>.
```
Comment (kbdharun) [inline pages.th/window/cmd.md:4]:
```suggestion
> ข้อมูลเพิ่มเติม: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/cmd>.
```

File: pages.th/window/dir.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# dir
+
+> แสดงรายการเนื้อหาไดเรกทอรี.
+> ข้อมูลเพิ่มเติม: <https://docs.microsoft.com/windows-server/administration/windows-commands/dir>.
```
Comment (kbdharun) [inline pages.th/window/dir.md:4]:
```suggestion
> ข้อมูลเพิ่มเติม: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/dir>.
```

File: pages.th/window/dir.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- Show the contents of the current directory:
+
+`dir`
+
+- แสดงเนื้อหาของไดเร็กทอรีปัจจุบัน:
+
+`dir {{path/to/directory}}`
```
Comment (kbdharun) [inline pages.th/window/dir.md:12]:
```suggestion
`dir {{ทาง\ไป\สารบบ}}`
```

File: pages.th/window/dir.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+
+- แสดงเนื้อหาของไดเร็กทอรีปัจจุบัน, รวมถึงรายการที่ซ่อนอยู่:
+
+`dir /A`
+
+- แสดงเนื้อหาของไดเร็กทอรีที่กำหนด, รวมถึงรายการที่ซ่อนอยู่:
+
+`dir {{path/to/directory}} /A`
```
Comment (kbdharun) [inline pages.th/window/dir.md:20]:
```suggestion
`dir {{ทาง\ไป\สารบบ}} /A`
```

**Review by kbdharun — APPROVED** [review]

File: pages.th/window/cmd.md [hunk 8]
```diff
@@ -0,0 +1,36 @@
+# cmd
+
+> ตัวแปลคำสั่งของ Windows.
```
Comment (MasterIceZ) [inline pages.th/window/cmd.md:3]:
```suggestion
> ตัวแปลคำสั่งของ Windows
```

File: pages.th/window/cd.md [hunk 9]
```diff
@@ -0,0 +1,24 @@
+# cd
+
+> แสดงชื่อหรือเปลี่ยนไดเร็กทอรีการทำงานปัจจุบัน.
```
Comment (MasterIceZ) [inline pages.th/window/cd.md:3]:
```suggestion
> แสดงชื่อหรือเปลี่ยนไดเร็กทอรีการทำงานปัจจุบัน
```

File: pages.th/window/dir.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+# dir
+
+> แสดงรายการเนื้อหาไดเรกทอรี.
```
Comment (MasterIceZ) [inline pages.th/window/dir.md:3]:
```suggestion
> แสดงรายการเนื้อหาไดเรกทอรี
```

**Review by MasterIceZ — COMMENTED** [review]
Anything else looks fine.

**Comment by kbdharun** [comment]
Thanks for the review, that was fast😄 .

**Comment by kbdharun** [comment]
Should we remove the period in other Thai translation pages too? @MasterIceZ

**Comment by MasterIceZ** [comment]
A period in Thai is optional, but in most of the sentences, we do not write it. So I think removing it might be a more native way to write.

File: pages.th/window/dir.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
+# dir
+
+> แสดงรายการเนื้อหาไดเรกทอรี
+> ข้อมูลเพิ่มเติม: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/dir>.
```
Comment (Leomotors) [inline pages.th/window/dir.md:4]:
```suggestion
> ข้อมูลเพิ่มเติม: <https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/dir>
```

**Review by Leomotors — COMMENTED** [review]
Everything else looks good for me :+1:

**Pull request merged**